src/components/DerivIframe/DerivIframe.tsx

```diff
@@ -1,7 +1,17 @@
 import { URLConstants } from '@deriv-com/utils';
 
 const DerivIframe = () => {
-    const origin = URLConstants.derivP2pProduction;
+    const getAllowedLocalStorageOrigin = () => {
+        if (/^staging-p2p\.deriv\.com$/i.test(window.location.hostname)) {
+            return URLConstants.derivP2pStaging;
+        } else if (/^p2p\.deriv\.com$/i.test(window.location.hostname)) {
+            return URLConstants.derivP2pProduction;
+        }
+        return URLConstants.derivP2pProduction;
```

Comment:
cant we just return URLConstants.deriv.P2pProduction since it looks like its the default one, correct me if im wrong

```suggestion
        } 
       
         return URLConstants.derivP2pProduction;
```

Reply:
for now it's fine. later if we have more envs we can add them .